WooCommerce/Classes/Authentication/Store Creation/Plan/WebPurchasesForWPComPlans.swift

```diff
@@ -54,6 +54,11 @@ extension WebPurchasesForWPComPlans: InAppPurchasesForWPComPlansProtocol {
         // Web purchases are available for everyone and every site.
         true
     }
+
+    func siteHasCurrentInAppPurchases(siteID: Int64) async -> Bool {
+        // no-op
+        false
```

Comment:
To consider for another PR: It might be worth giving the real answer here, i.e. checking for the SiteID in this user's transactions. Or to error in some way – the site _might_ have an In-App Purchase, even if you shouldn't be able to get to the view now. 

If the implementation is the same you could make it a protocol extension.

Reply:
That sounds good. Opened a follow-up issue [here](https://github.com/woocommerce/woocommerce-ios/issues/10335), since I'm not sure if we're removing the `WebPurchasesForWPComPlans` feature in favour of IAP-only 🤔